src/assets/js/WhoCallsTheFleetItemDb.json

```diff
@@ -138,17 +138,17 @@
 {"id":21,"rarity":1,"type":18,"name":{"ja_jp":"零式艦戦52型","ja_kana":"","ja_romaji":"","zh_cn":"零式舰战52型"},"stat":{"fire":0,"torpedo":0,"bomb":0,"asw":0,"aa":6,"armor":0,"evasion":0,"hit":0,"los":0,"range":0,"distance":6},"dismantle":[1,2,0,3],"default_equipped_on":[285,291,296,292,297,318,281,282,117,283,284,277,278,279,280,288,112],"time_created":1427787070947,"_id":"e7wSryHFqgoTPPMb","craftable":true,"improvable":true,"time_modified":1471227157203,"upgrade_from":[181],"rankupgradable":"on","improvement":[{"upgrade":false,"req":[[[false,false,false,false,true,true,true],[110,288,461,466]],[[true,true,false,true,false,false,false],[111,112,462,467]]],"resource":[[120,120,0,280],[3,5,3,4,21,1],[5,8,3,5,21,2],[0,0,0,0,0,0]]}],"upgrade_for":[60,198,153,154,156,152,157,109,205]}
 {"id":162,"rarity":3,"type":4,"name":{"ja_jp":"203mm/53 連装砲","ja_kana":"","ja_romaji":"","zh_cn":"203mm/53 连装炮"},"stat":{"fire":9,"torpedo":0,"bomb":0,"asw":0,"aa":1,"armor":0,"evasion":0,"hit":-2,"los":0,"range":3,"distance":3},"dismantle":[0,4,4,0],"craftable":false,"improvable":true,"time_created":1455111285227,"_id":"eMfXN9ND3e3dUkbJ","default_equipped_on":[448,358,496,449,361],"time_modified":1488827702107,"improvement":[{"upgrade":false,"req":[[[true,true,true,true,false,false,false],[496]]],"resource":[[10,180,190,0],[4,5,3,4,6,2],[5,8,4,7,90,1],[0,0,0,0,0,0]]}]}
 {"id":69,"rarity":3,"type":22,"name":{"ja_jp":"カ号観測機","ja_kana":"","ja_romaji":"","zh_cn":"KA号观测机"},"stat":{"fire":0,"torpedo":0,"bomb":0,"asw":9,"aa":0,"armor":0,"evasion":0,"hit":1,"los":0,"range":0,"distance":4},"dismantle":[1,1,0,4],"default_equipped_on":[166],"time_created":1427788258669,"_id":"ewtWFI9jWb0UIGqd","time_modified":1471227157618}
-{"id":215,"rarity":4,"type":51,"name":{"ja_jp":"Ro.44水上戦闘機bis","ja_kana":"","ja_romaji":"","zh_cn":"Ro.44水上战斗机bis"},"stat":{"fire":0,"torpedo":0,"bomb":0,"asw":1,"aa":3,"armor":0,"evasion":2,"hit":2,"los":3,"range":"","distance":3},"dismantle":[1,1,0,2],"rankupgradable":"on","craftable":false,"improvable":true,"improvement":[{"upgrade":false,"req":[[[false,false,false,false,false,true,true],[496]]],"resource":[[80,60,-1,280],[8,-1,4,-1,0,0],[-1,-1,-1,-1,0,0],[0,0,0,0,0,0]]}],"time_created":1489771924871,"_id":"fAZlB7RCNRBRRum3","time_modified":1489775806628,"upgrade_from":[164]}
+{"id":215,"rarity":4,"type":51,"name":{"ja_jp":"Ro.44水上戦闘機bis","ja_kana":"","ja_romaji":"","zh_cn":"Ro.44水上战斗机bis"},"stat":{"fire":0,"torpedo":0,"bomb":0,"asw":1,"aa":3,"armor":0,"evasion":2,"hit":2,"los":3,"range":"","distance":3},"dismantle":[1,1,0,2],"rankupgradable":"on","craftable":false,"improvable":true,"improvement":[{"upgrade":false,"req":[[[false,false,false,false,false,true,true],[496]]],"resource":[[80,60,0,280],[8,14,4,5,26,2],[14,-1,6,-1,164,1],[0,0,0,0,0,0]]}],"time_created":1489771924871,"_id":"fAZlB7RCNRBRRum3","time_modified":1490201028477,"upgrade_from":[164]}
```

Comment:
dont do it at this side, report the issue to src reop. there are other wrong things i know 😂